pages/common/npm-update.md

```diff
@@ -0,0 +1,20 @@
+# npm update
+
+> Update a package or packages in the current project. 
+> More information: <https://docs.npmjs.com/cli/v8/commands/npm-update>.
+
+- Update all packages in the current project:
+
+`npm update`
+
+- Update a specific package in the current project:
+
+`npm update {{package_name}}`
+
+- Update a package globally:
+
+`npm update -g {{package_name}}`
+
+- Update multiple packages at once:
+
+`npm update {{package_name1 package_name2 ...}}`
```

Comment:
Thanks for your contribution, I have a suggestion for this page.

```suggestion
`npm update {{package}}`

- Update a package globally:

`npm update -g {{package}}`

- Update multiple packages at once:

`npm update {{package1 package2 ...}}`
```
We can use `package` instead `package_name` here to align with other package manager pages.